content/updates/04-2020/3-connection-dashboard-updates.md

```diff
@@ -1,25 +1,24 @@
 # Connection dashboard updates
-::: meta-data type="New features" features="Connections" available="2nd of April"
+::: meta-data type="New" features="Connections, Connectivity" available="2nd of April"
 :::
 
-## Source connections charts rework
+## Source connections charts improved
 We reworked the display of the two charts about the source connections in the Connection dashboard. We also now compute the sum on the last 7 days of the number of products created and updated.
 
 ![Sources connection charts](../img/number-of-products-created-updated.png)
 
 ## Track your destination connections
-You can now track the number of products that were sent to your destinations in the Connection dashboard. You can consult this metric for each of the last 7 days. We also compute the sum of the last 7 days.
+You can now easily track the number of products that were sent to your destinations in the Connection dashboard. You can consult this metric for each of the last 7 days. We also compute the sum of the last 7 days.
 
 ![Destination connection charts](../img/number-of-products-sent.png)
 
-## Choose which connection you want to track
```

Comment:
I wouldn't remove this title. This is a new feature on itself.
It's missing meta data as well.